tests/scripts/run-xunit-perf.cmd

```diff
@@ -70,6 +78,23 @@ setlocal ENABLEDELAYEDEXPANSION
 
   exit /b %ERRORLEVEL%
 
+:run_slice_benchmarks
+setlocal 
+  set /P CORECLR_TEST_DIR=< %~1
+  echo %CORECLR_TEST_DIR%
+  set CORECLR_TEST_FULL_PATH=%CORECLR_PERF%%CORECLR_TEST_DIR%
+  echo %CORECLR_TEST_FULL_PATH%
+  for /R %CORECLR_TEST_FULL_PATH% %%T in (*.%TEST_FILE_EXT%) do (
+    call :run_benchmark %%T || (
+      set /A "LV_FAILURES+=1"
+    )
+  )
+
+  del %~1
```

Comment:
>del %~1 [](start = 2, length = 7)

Where is this file generated? I would not delete it here, but where it was created.